docs/src/main/asciidoc/spring-cloud-commons.adoc

```diff
@@ -1,12 +1,13 @@
 = Cloud Native Applications
-:github: https://github.com/spring-cloud/spring-cloud-commons
+:githubroot: https://github.com/spring-cloud
+:github: {githubroot}/spring-cloud-commons
 :githubmaster: {github}/tree/master
 :docslink: {githubmaster}/docs/src/main/asciidoc
 :toc:
 
 include::intro.adoc[]
 
-include::https://raw.githubusercontent.com/spring-cloud/spring-cloud-build/master/docs/src/main/asciidoc/contributing-docs.adoc[]
+include::https://github.com/spring-cloud/spring-cloud-build/blob/master/docs/src/main/asciidoc/contributing-docs.adoc[]
```

Comment:
Not sure why this doesn't get included. The `blob` link renders a github page so if it does get included it would break.